glusterd2/commands/volumes/bricks-status.go

```diff
@@ -2,28 +2,43 @@ package volumecommands
 
 import (
 	"net/http"
-	"strings"
-	"syscall"
 
 	"github.com/gluster/glusterd2/glusterd2/brick"
-	"github.com/gluster/glusterd2/glusterd2/daemon"
 	"github.com/gluster/glusterd2/glusterd2/gdctx"
-	"github.com/gluster/glusterd2/glusterd2/pmap"
 	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
 	"github.com/gluster/glusterd2/glusterd2/transaction"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/pkg/api"
 	"github.com/gluster/glusterd2/pkg/errors"
-
 	"github.com/gorilla/mux"
 )
 
 const (
 	brickStatusTxnKey string = "brickstatuses"
 )
 
-func checkBricksStatus(ctx transaction.TxnCtx) error {
+func registerBricksStatusStepFuncs() {
+	transaction.RegisterStepFunc(bricksStatus, "bricks-status.Check")
+}
+func createBrickStatusRspAPI(brickStatuses []brick.Brickstatus) []*api.BrickStatus {
```

Comment:
The name will be slightly better if this is called `createBrickStatusResp` to be consistent with rest of the code.